tests/integration-tests/chaos-experiments/helpers/requestCreator.js

```diff
@@ -16,13 +16,146 @@ module.exports = {
 
 async function init() {
     try {
-        app = await expressApp();
+        nockK8sChaosExperimentSupportedKinds();
+        const appInitUtils = require('../../testUtils');
+        app = await appInitUtils.getCreateTestApp();
     } catch (err){
         console.log(err);
         process.exit(1);
     }
 }
 
+function nockK8sChaosExperimentSupportedKinds(url) {
+    const response = {
+        kind: 'CustomResourceDefinitionList',
+        apiVersion: 'apiextensions.k8s.io/v1',
+        metadata: {
+            resourceVersion: '123456'
+        },
+        items: [
```

Comment:
you don't really need to have this long object.. we only use one field here which is the name. you can have a short list with only the properties we need.

Reply:
shortened